storage/namespace.go

```diff
@@ -277,6 +346,16 @@ func (n *dbNamespace) Bootstrap(
 	workers := pool.NewWorkerPool(int(math.Ceil(float64(runtime.NumCPU()) / 2)))
```

Comment:
might make sense to reuse this by making it a namespace field if we might bootstrap multiple times